example/example.dart

```diff
@@ -7,40 +7,46 @@ import 'dart:io';
 
 import 'package:crypto/crypto.dart';
 
-final _usage = 'Usage: dart hash.dart <md5|sha1|sha256> <input_filename>';
+final _usage = 'Usage: dart example.dart <md5|sha1|sha256> <input_filename>';
 
 Future main(List<String> args) async {
   if (args == null || args.length != 2) {
     print(_usage);
     exit(1);
   }
 
-  Hash hasher;
+  var hasher = _getHasher(args[0]);
```

Comment:
What's with the term "hasher"? Why not just use "hash", since that's what the class is?

Reply:
Zero clue. Just trying to minimize code churn

Reply:
Since you're making changes here, can you fix it?